docs/configuration.md

```diff
@@ -4,12 +4,10 @@ Since the server implementation is built with [FastAPI](https://fastapi.tiangolo
 This way of handling configuration options supports various different approaches to configure the server.
 We recommend either or a combination of the following:
 
-1. Set environment variables.
-2. Create a JSON file with an implementation's complete configuration.
-
-When adding a third, final option, this also represents the (descending) order of priority with which configuration values are determined:
-
-- Use defaults in [`ServerConfig`][optimade.server.config.ServerConfig].
+1. Create a custom [`ServerConfig`][optimade.server.config.ServerConfig] object with the desired settings directly.
+2. Set environment variables prefixed with `OPTIMADE_` or `optimade_`.
+3. Create a JSON or YAML configuration file with an implementation's complete configuration in the default location [DEFAULT_CONFIG_FILE_PATH][optimade.server.config.DEFAULT_CONFIG_FILE_PATH] or specify its location with the `OPTIMADE_CONFIG_FILE` environment variable.
```

Comment:
```suggestion
3. Create a JSON or YAML configuration file with an implementation's complete configuration in the default location [DEFAULT_CONFIG_FILE_PATH][optimade.server.config.DEFAULT_CONFIG_FILE_PATH] or specify its absolute path with the `OPTIMADE_CONFIG_FILE` environment variable.
```

Reply:
Disagree on this one just because I sometimes use relative paths, and its useful if you are transporting stuff around without containerizing

Reply:
I'm unsure whether that works though? Hmm, I guess it works from where ever you're current working directory is when you run the server?

Reply:
It does work, plus we have guards in place that check whether the file exists. I think allowing both kinds of path is fine

Reply:
It sure is - I've just added it to my incoming commit under the appropriate section.